crates/blockchain/mempool.rs

```diff
@@ -54,31 +54,26 @@ pub fn add_transaction(transaction: Transaction, store: Store) -> Result<H256, M
     Ok(hash)
 }
 
-/// Fetch a transaction from the mempool
-pub fn get_transaction(
-    hash: H256,
-    store: Store,
-) -> Result<Option<MempoolTransaction>, MempoolError> {
-    Ok(store.get_transaction_from_pool(hash)?)
-}
-
 /// Fetch a blobs bundle from the mempool given its blob transaction hash
 pub fn get_blobs_bundle(tx_hash: H256, store: Store) -> Result<Option<BlobsBundle>, MempoolError> {
-    Ok(store.get_blobs_bundle_from_pool(tx_hash)?)
+    store
+        .get_blobs_bundle_from_pool(tx_hash)
+        .map_err(MempoolError::from)
 }
 
 /// Applies the filter and returns a set of suitable transactions from the mempool.
-/// These transactions will be grouped by sender and sorted by nonce
+/// These transactions will be grouped by sender and sorted by hash.
```

Comment:
sorted by hash doesn't make much sense IMO.

Reply:
This has been updated.